docs/4.x/piwiks-ini-configuration.md

```diff
@@ -64,3 +64,11 @@ Config::getInstance()->forceSave();
 **Plugins cannot add new configuration options.** If you are creating a core contribution and want to add a new INI option, you can simply add the option and its default value to `global.ini.php`.
 
 If you want to make your plugin configurable, create a [Plugin Setting](/guides/plugin-settings).
+
+## Boolean Configuration Options
+
+For example `force_ssl = 1` is a boolean value in the configuration. By convention we use either 0 or 1 exactly for the value. It was suggested that we improve and clarify this in the future.
```

Comment:
could add a link here? `[improve and clarify this in the future](https://github.com/matomo-org/matomo/issues/17876).` --> That's the issue I created for it.